packages/teams-js/src/public/authentication.ts

```diff
@@ -668,6 +668,34 @@ export namespace authentication {
      * Limited to Microsoft-internal use
      */
     ver: string;
+    /**
+     * @hidden
+     * Stores the residency of the user.
+     *
+     * @internal
+     * Limited to Microsoft-internal use
+     */
+    dataResidency: DataResidencySlim;
+  }
+
+  /**
+   * Limited set of data residencies information exposed to 1P application developers
+   */
+  export enum DataResidencySlim {
```

Comment:
>DataResidencySlim [](http://example.com/codeflow?start=14&length=17)

Why does this type have "slim" in the name?

Reply:
We used the name `DataResidency` on the host side for an enum that has more values than are being exported publicly. The internal-only `DataResidency` enum values are mapped to the public-facing ones in `DataResidencySlim`.

Reply:
Is that host implementation detail (that it's the "slim" version) something that is important for consumers of teamsjs be aware of?

Reply:
I think this should just be `DataResidency` here in TeamsJs.